src/cryptonote_core/service_node_rules.cpp

```diff
@@ -77,11 +77,8 @@ uint64_t get_staking_requirement(cryptonote::network_type m_nettype, uint64_t he
   uint64_t base = 0, variable = 0;
   if (hf_version >= cryptonote::network_version_11_infinite_staking)
   {
-    auto round_method = std::fegetround();
-    std::fesetround(FE_TONEAREST);
```

Comment:
I think this should stay: I don't think there's any particular guarantee as to what rounding mode the thread is in without an explicit call, and I'm assuming it's essentially a no-op if setting it to the mode it's already in.

Reply:
And updated.